components/NavigationDrawerListItem/NavigationDrawer/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import Image from 'next/image'
+// MUI //
+import Drawer from '@mui/material/Drawer'
+import { Box } from '@mui/system'
+import Collapse from '@mui/material/Collapse'
+// other mui dependencies
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+// icons and text
+// import Typography from '@mui/material/Typography'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import SettingsIcon from '@mui/icons-material/SettingsOutlined'
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import UnarchiveOutlinedIcon from '@mui/icons-material/UnarchiveOutlined'
+import AssignmentLateOutlinedIcon from '@mui/icons-material/AssignmentLateOutlined'
+import AssignmentOutlinedIcon from '@mui/icons-material/AssignmentOutlined'
+// List component
+import NavigationDrawerListItem from '..'
+// List components from MUI
+import List from '@mui/material/List'
+import ListItemText from '@mui/material/ListItemText'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemButton from '@mui/material/ListItemButton'
+
+interface NavigationDrawerProps {
+  open: boolean
+}
+export default function NavigationDrawer({
+  open = true,
+}: NavigationDrawerProps) {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
+      style={{ objectFit: 'contain' }}
+    />
+  )
+  // state handlers for expandable lists in drawer
+  const [CollapseOpen, setOpen] = React.useState(true)
+  const handleClick = () => {
+    setOpen(!CollapseOpen)
+  }
+  // Drawer setup
+  return open ? (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        variant="permanent"
+        sx={{
+          width: drawerWidth,
```

Comment:
Add a prop here `onClose` that calls `setDrawerOpen` with false:

```jsx
<Drawer ... onClose={() => setDrawerOpen(false)> ... </Drawer>
```

this will let the drawer close when someone clicks away